Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -221,11 +244,13 @@ private sealed class UnstickSuccessfulEvent : EntityEventArgs
     {
         public readonly EntityUid Uid;
         public readonly EntityUid User;
+        public readonly EntityUid Target;
```

Comment:
It's redundant, because you have `component.StuckTo` field.

Reply:
Yes I am working on replacing all of the checks with it